src/rules/converters/space-before-function-paren.ts

```diff
@@ -1,29 +1,32 @@
 import { RuleConverter } from "../converter";
 
-const NOT_SUPPORTED_OPTIONS = ["constructor", "method"];
-
-function isExistedESLintOption(rule: string) {
-    return !NOT_SUPPORTED_OPTIONS.includes(rule);
-}
+const SUPPORTED_OPTIONS: string[] = ["anonymous", "asyncArrow", "named"];
 
 export const convertSpaceBeforeFunctionParen: RuleConverter = tslintRule => {
-    const ruleArguments = tslintRule.ruleArguments.filter(isExistedESLintOption);
-    const notices = NOT_SUPPORTED_OPTIONS.reduce<string[]>((acc, option) => {
-        if (tslintRule.ruleArguments.includes(option)) {
-            acc.push(`Option "${option}" is not supported by ESLint.`);
-        }
-        return acc;
-    }, []);
-
-    return {
-        rules: [
-            {
-                ...(tslintRule.ruleArguments.length !== 0 && {
-                    ruleArguments,
+    const argument: "always" | "never" | { [key: string]: "always" | "never" } | undefined =
+        tslintRule.ruleArguments[0];
+    const ruleName = "space-before-function-paren";
+    if (argument === "always" || argument === "never") {
+        return { rules: [{ ruleArguments: [argument], ruleName }] };
```

Comment:
I'm going to send a separate change to enable [`no-else-return`](https://eslint.org/docs/rules/no-else-return) in the master branch. Could you please fix its complaints here? That way we don't get surprises when the PR is merged.

Reply:
OK. I will fix it.